src/inference/src/threading/ie_cpu_streams_executor.cpp

```diff
@@ -98,7 +98,16 @@ struct CPUStreamsExecutor::Impl {
                     // wrapping around total_streams (i.e. how many streams all different core types can handle
                     // together)
                     const auto total_streams = _impl->total_streams_on_core_types.back().second;
+                    const auto hybrid_core = _impl->total_streams_on_core_types.size() > 1;
```

Comment:
Will double check if `_impl->total_streams_on_core_types.size()` will be impacted by numactl